appveyor.yml

```diff
@@ -1,7 +1,7 @@
 platform:
   - x64
 
-image: Visual Studio 2015
+image: Visual Studio 2022
```

Comment:
See the `ARTEFACTS: yes` case below... this will almost certainly need to use an old image to maintain the binary releases as they are presently done (see #41 and #46).

Given that VS 2015 is still supported, I'm not sure that the switch to VS 2022 is worth it?

Reply:
You're likely right, I'm reverting that part.